Cargo.toml

```diff
@@ -15,7 +15,8 @@ description = "An async extension for Diesel the safe, extensible ORM and Query
 [dependencies]
 diesel = { version = "2.0.0", default-features = false, features = ["i-implement-a-third-party-backend-and-opt-into-breaking-changes"]}
 async-trait = "0.1.51"
-futures = "0.3.17"
+futures-channel = { version = "0.3.17", default-features = false, features = ["std", "sink"] }
```

Comment:
Channels are currently only used by the mysql connection implementation. It would be meaningful to mark this dependency as optional and put it behind the `mysql` feature flag.